src/Tec/Plugin.php

```diff
@@ -144,43 +148,32 @@ function option_filter( $fields ) {
 			'3|day'     => esc_html__( '3 days', 'the-events-calendar' ),
 			'1|week'    => esc_html__( '1 week', 'the-events-calendar' ),
 			'2|week'    => esc_html__( '2 weeks', 'the-events-calendar' ),
-			'1|month'   => esc_html__( '1 month', 'the-events-calendar' ),
-			'3|month'   => esc_html__( '3 months', 'the-events-calendar' ),
-			'6|month'   => esc_html__( '6 months', 'the-events-calendar' ),
-			'9|month'   => esc_html__( '9 months', 'the-events-calendar' ),
-			'1|year'    => esc_html__( '1 year', 'the-events-calendar' ),
-			'2|year'    => esc_html__( '2 years', 'the-events-calendar' ),
-			'3|year'    => esc_html__( '3 years', 'the-events-calendar' ),
+			'1'         => esc_html__( '1 month', 'the-events-calendar' ),
+			'3'         => esc_html__( '3 months', 'the-events-calendar' ),
+			'6'         => esc_html__( '6 months', 'the-events-calendar' ),
+			'9'         => esc_html__( '9 months', 'the-events-calendar' ),
+			'12'        => esc_html__( '1 year', 'the-events-calendar' ),
+			'24'        => esc_html__( '2 years', 'the-events-calendar' ),
+			'36'        => esc_html__( '3 years', 'the-events-calendar' ),
 		];
 		$fields['delete-past-events']['options'] = $new_values;
 		$fields['trash-past-events']['options']  = $new_values;
```

Comment:
Lets delete what we don't interfere with.

Reply:
Removed.